src/rules/javascript/webextension-unsupported-api.js

```diff
@@ -5,7 +5,8 @@ export default {
   create(context) {
     return {
       MemberExpression: function(node) {
-        if (node.object.object &&
+        if (!node.computed &&
```

Comment:
As far as actually determining the value that would likely involve running the code.

In the case that prompted this we would be able to do it (a const assigned at top of file and used throughout) but that's a special case that could be handled with static analysis (in theory).

If it was instead pulling the value from a user-managed variable or generating a random string for some reason we'd probably have no possible way of knowing what value or values it would be.